views/index.ejs

```diff
@@ -1,10 +1,44 @@
 <% include partials/header %>
+<div class="FlagImgContainer"></div>
+<div class="RedBackgroundContainer">
+  <div class="TinyLogo TinyLogoLandingPage"></div>
+  <h1 class="Display1 Display3 WhiteText">
+    Do you know who represents you?
+  </h1>
+  <form id="zipForm">
+    <label class="WhiteText">
+      Enter your email
+    </label>
+    <input type="tel" name="zip" id="zipInput" value="98144" class="InputZip">
+    <input type="submit" value="Go" class="SubmitZip">
+  </form>
+  <div class="LandingPageQuote WhiteText">
+    <p class="LandingPageQuoteSpan">
+      &ldquo;When the people become involved&hellip; government becomes more accountable, and our society is stronger.&rdquo;
+    </p>
+    <p class="mt4">
+      &ndash;Arnold Schwarzenegger
+    </p>
+  </div>
+  <p class="BuiltBy">
+    Built by <a href="https://twitter.com/jeremypbeasley"><span class="BlackText BorderBottomBlack">@jeremypbeasley</span></a> and <a href="https://twitter.com/jennminerva"><span class="BlackText BorderBottomBlack">@jennminerva</span></a>
```

Comment:
add `target="_blank` on all three of these links. https://www.w3schools.com/tags/att_a_target.asp We wanna keep people on the site. If you're leaving the domain, it's considered customary to use a new tab.